tests/test_server.py

```diff
@@ -180,8 +182,32 @@ def test_list_all_prod_info(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual("Inventory is empty.", response.data)        
 
-    def test_update_prof_info(self):
+    def test_restock_action(self):
+        # Test restocking a non-existing product.
+        response = self.app.put(PATH_RESTOCK.format(3), data = json.dumps({RESTOCK_AMT: 43}),
```

Comment:
Styling issue: data=json.dump... (no space on two sides of '=' inside function). Same in line 188, 192, 197.